src/components/shared/Typography/Body1.jsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+export default styled.p`
+  font-style: ${(props) => (props.bold ? 'bold' : 'normal')};
+  ${tw`
+   font-normal
```

Comment:
The font family is missing everywhere, please check the figma file and include the required font. Let me know if you need help with this.